### PR TITLE
bumpup dev branch

### PR DESCRIPTION
**Pull request by LianaHus**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "remix-project",
-  "version": "0.10.5",
+  "version": "0.10.6-dev",
```
Comment (yann300) [inline package.json:3]: This should be `"version": "0.10.7-dev",`

File: package-lock.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "remix-project",
-  "version": "0.10.5",
+  "version": "0.10.6-dev",
```
Comment (Aniket-Engg) [inline package-lock.json:3]: This also needs to be updated

**Review by cleytontiagosorinho — APPROVED** [review]

**Review by cleytontiagosorinho — APPROVED** [review]

**Review by cleytontiagosorinho — APPROVED** [review]
bumpDev

**Review by yann300 — CHANGES_REQUESTED** [review]
`"version": "0.10.6-dev",` need to be updated in the lock file

**Review by yann300 — APPROVED** [review]

**Pull request merged**